src/api/entities/CheckpointSchedule/index.ts

```diff
@@ -112,6 +114,17 @@ export class CheckpointSchedule extends Entity<UniqueIdentifiers> {
 
     const rawSchedules = await checkpoint.schedules(stringToTicker(ticker, context));
 
+    const scheduleIds = rawSchedules.map(({ id: scheduleId }) =>
```

Comment:
You can just do `find` here and save yourself having to go through the entire array converting everything

Reply:
Same for the `exists` method itself